### ISSUE TITLE
Button id is cannot pass

### ISSUE THREAD

**Issue by phucanthony** [cloudle/ruui#7]
```
<Button id="someid"/>
```

in test cannot recognize the id. Same for testID and accessibilityLabel

**Comment by phucanthony** [comment]
same for dropdownContainer

**Comment by cloudle** [comment]
@phucanthony I've just updated.. add 9 props for Both accessibility/test stuffs:

- id?: string
- nativeID?: string
- testID?: string
- accessible?: boolean
- accessibilityLabel?: any
- accessibilityComponentType?: AccessibilityComponentType
- accessibilityTraits?: AccessibilityTrait
- onAccessibilityTap?: Function
- onMagicTap?: Function
